### ISSUE TITLE
NVDA should read all progress bars in the foreground window

### ISSUE THREAD

**Issue by mdlawler** [nvaccess/nvda#18372]
### Is your feature request related to a problem? Please describe.
no
### Describe the solution you'd like
I'd like to see NVDA read all progress bars in the foreground window.  I've seen several installers, but I can't currently think of an example, that have multiple progress bars.  Also for the progress bars that just move across the screen from left to right over and over I believe NVDA should say "progressing" for these.
### Additional context
This was how Window-Eyes worked.

**Comment by Adriani90** [comment]
Actually the installer progress bar is being announced properly by NVDA. If this is not exposed properly by the accessibility API, then NVDA users at least can access the object with a progressing property via object navigation.
I am closing as works for me.
Please provide an example application where you can reproduce this, and provide the exact steps you tried in order to read the progress bars. If this is a valdi problem, we can reopen.

**Comment by mdlawler** [comment]
I see this with the Icewarp mail server installer  at https://dl.icewarp.com/server/Windows/icewarp14/IceWarpServer-14.2.0_x64.exe during the install.  There are three progress bars, but NVDA only reports one of them.  I also see this while running a backup with Casper from http://fssdev.com as there are two progress bars and NVDA only reports one of them.  This doesn't work correctly and should be reopened.

**Comment by gerald-hartig** [comment]
NVDA should be reading all progress bars.

**Comment by mdlawler** [comment]
I agree, but it doesn't seem to be doing so with the software I mentioned above.  Window-Eyes also numbered them to indicate that more than one was being read concurrently so "progress 1 25 percent" "progress 2 37 percent" "progress 3 12 percent" and then each time one changed WE would say "progress x percent x".  If this isn't done the user has no idea how many progress bars are being displayed at a time.  From what I can tell bye the way NVDA is reading and knowing how the progress bars in the above mentioned software work I believe that it is only reading one of the bars.
